CRM/Contact/BAO/Group.php

```diff
@@ -384,6 +384,22 @@ public static function &create(&$params) {
       $params['modified_id'] = $cid;
     }
 
+    // CRM-19068.
+    // Validate parents parameter when creating group.
+    if (isset($params['parents'])) {
+      if (is_array($params['parents'])) {
+        foreach ($params['parents'] as $parent => $dc) {
+          if (!CRM_Utils_Type::validate($parent, 'Integer', FALSE)) {
+            unset($params['parents'][$parent]);
```

Comment:
@totten the only reason is that I think this maybe used by the add group form as well but i guess maybe yeh we should chuck error instead.